MyoSimulatorForm/MyoSimulatorForm/ParsedCommands/ParsedCommand.cs

```diff
@@ -16,9 +16,8 @@ public enum CommandType { MOVE, SET_ACCELERATION, ASYNC, EXPECT, PADDING }
         // Note: Arm Sync/Unsync is not listed here, because Arm Sync requires additional
         // parameters. It should either be its own command, or we should allow optional extra
         // parameters for some async commands.
-        /* NOTE: Do not remove RESERVED1. It is needed to preserved the order of the enum */
         public enum AsyncCommandCode { PAIR, UNPAIR, CONNECT, DISCONNECT, ARM_RECOGNIZED, ARM_LOST, REST, FIST, 
-            WAVE_IN, WAVE_OUT, FINGERS_SPREAD, THUMB_TO_PINKY, RESERVED1, UNKNOWN};
+            WAVE_IN, WAVE_OUT, FINGERS_SPREAD, THUMB_TO_PINKY, UNKNOWN};
 
```

Comment:
As far as I can tell, it's ok to remove RESERVED1 here because the problem was when the MyoSim sends an async command number to the hub, the number sent does not match the correct command in the Hub. I.e. the HubCommunicator must have the same enum as the Hub to send the correct number, but removing it here just removes the ability for MyoSim to use RESERVED1 internally.